docs/user_management/security_privacy.md

```diff
@@ -60,6 +60,10 @@ A compromised agbot can attempt to propose malicious workload deployments, but t
 
 {{site.data.keyword.horizon_exchange}} is a centralized, replicated, and load balanced REST API server. It functions as a shared database of metadata for users, organizations, edge nodes, published services, policies, and patterns. It also enables the distributed agents and agbots to deploy containerized workloads by providing the storage for the secure control plane, until the messages can be retrieved. The {{site.data.keyword.horizon_exchange}} is unable to read the control messages because it does not possess the private RSA key to decrypt the message. Thus a compromised {{site.data.keyword.horizon_exchange}} is incapable of spying on the control plane traffic. For more information on the role of the exchange, see [Overview of {{site.data.keyword.edge}}](../getting_started/overview_ieam.md).
 
+## Privileged mode services
+{: #priv_services}
+On a host machine, some tasks can only be performed by an account with root access. The equivalent for containers is privileged mode. While containers generally do not need privileged mode on the host, there are some use cases where it is required. In {{site.data.keyword.ieam}} you have the ability to specify that an application service should be deployed with privileged process execution enabled. By default, it is disabled. You must explicitly enable it in the [deployment configuration](https://github.com/open-horizon/anax/blob/master/docs/deployment_string.md) of the respective Service Definition file for each service that needs to run in  this mode. And further, any node on which you want to deploy that service must also explicitly allow privileged mode containers. This ensures that node owners have some control over which services are executing on their edge nodes. For an example of how to enable privileged mode policy on an edge node, see [privileged node policy](https://github.com/open-horizon/anax/blob/master/cli/samples/privileged_node_policy.json). If the service definition or one of its dependencies requires privileged mode, the node policy must also allow privileged mode, or else none of the services will not be deployed to the node. For an indepth discussion of privileged mode see [What is privileged mode and do I need it?](https://wiki.lfedge.org/pages/viewpage.action?pageId=44171856).
```

Comment:
The "deployment configuration" URL is https://open-horizon.github.io/anax/deployment_string.html

Reply:
done